elections.js

```diff
@@ -8,6 +8,24 @@ const client = new Orbs.Client("http://3.134.6.50/vchains/1100000", virtualChain
 const orbsVotingContractName = "_Elections";
 let currentElectionsData;
 
+const initElectionsDataPromise = new Promise((res, rej) => {
+    let initPid = setInterval(() => {
+        if (currentElectionsData && currentElectionsData.ok) {
+            clearInterval(initPid);
+            res(currentElectionsData);
+        }
+    }, 50);
+
+    // Protection in case, we get no answer in 5 seconds
+    setTimeout(() => {
```

Comment:
I would:
1. set the timeout for far larger than 5 seconds, at least 25 seconds. 
1. consider seriously just removing this clause without addressing this edge case
1. if I would retain it - i would merge this logic with the above function - every iteration we would check the accumulated time passed during the polling and if it is > 25 seconds, then we  `res({ok:false, message:'...failed!'})` - this way we always  `res` and stop the interval.

please notice i think you were wrong to `rej` here - you should have `res` because this would cause the handler to throw an exception and never respond to the request - well, it would depend on how you would have handled the promise there.
but you forgot to add `await` or `.then().catch()` in the handler side...